src/components/AuthenticatedAppHeader/NoticiationPaneDisplayText.jsx

```diff
@@ -6,24 +6,37 @@ import Link from '../Link';
 
 export default function NotificationPaneDisplayText({
   currentNotificationSchema,
-  userName,
+  usrName,
```

Comment:
typo

Reply:
Intentional. The username prop is composed in this component; it consists of usrName and userNameGuid. Had to change to avoid scoping issues I think:
```
userName: (
          <span>
            <Link newTab href={`/users/${userNameGuid}`}>
              {usrName}
            </Link>
          </span>
        ),
```